### PR TITLE
fix: add rate limit for SMTP settings

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

— rate limit setting added to Auth Config settings for SMTP

### CODE REVIEW

File: studio/components/interfaces/Auth/SmtpForm/SmtpForm.tsx [hunk 1]
```diff
@@ -244,6 +244,15 @@ const SmtpForm = () => {
...
                     descriptionText="How long between each email can a new email be sent via your SMTP server."
                   />
+                  <Input
+                    type="number"
+                    name="RATE_LIMIT_EMAIL_SENT"
+                    id="RATE_LIMIT_EMAIL_SENT"
+                    min={0}
+                    label="Rate limit for sending emails"
+                    descriptionText="How many emails can be sent per hour."
+                    actions={<span className="text-scale-900 mr-3">per hour</span>}
```
Comment (joshenlim) [inline studio/components/interfaces/Auth/SmtpForm/SmtpForm.tsx:254]: could we shift this to `labelOptional="Per hour"` - for consistency and also cause the input fields vanilla increment/decrement button overlays this text

Comment (MildTomato) [reply studio/components/interfaces/Auth/SmtpForm/SmtpForm.tsx:254]: shall i just fix it in UI?

Comment (MildTomato) [reply studio/components/interfaces/Auth/SmtpForm/SmtpForm.tsx:254]: the default arrows need to be removed actually

**Review by joshenlim — APPROVED** [review]

File: studio/components/interfaces/Auth/SmtpForm/SmtpForm.tsx (hunk 1 above)
Comment (MildTomato) [reply studio/components/interfaces/Auth/SmtpForm/SmtpForm.tsx]: done

**Pull request merged**